.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.0.0a
```

Comment:
Actually, I recommend we keep `1.0.0a`. It's a way to indicate it's not a real release.

Reply:
It's a quick fix torelease 1.5.4 version, we will revert back the changes later.